dist/groucho.js

```diff
@@ -65,8 +65,11 @@ var groucho = window.groucho || {};
 
     // Log all configured items.
     // @todo Allow toggling tracking on and off as a localStorage help config override.
-    if (typeof trackExtent !== false && typeof dataLayer !== 'undefined') {
```

Comment:
It's impossible for a `typeof` expression to return the value `false` is what I was getting at.